ui/tests/acceptance/allocation-detail-test.js

```diff
@@ -153,10 +153,11 @@ module('Acceptance | allocation detail', function(hooks) {
       jobId: job.id,
     });
 
-    const taskState = allocation.task_states.models[0];
-    const task = server.schema.tasks.findBy({ name: taskState.name });
-    task.update('kind', 'connect-proxy:task');
-    task.save();
+    allocation.task_states.models.forEach(taskState => {
+      const task = server.schema.tasks.findBy({ name: taskState.name });
+      task.update('kind', 'connect-proxy:task');
+      task.save();
+    });
```

Comment:
I'm cool with this, but it seems like the thing that was biting you is that the tasks table is sorted alphabetically by task name. So if you wanted to stick with the original plan of only having one task be the proxy, you could mimic the sort by name behavior.

Your call!

Reply:
You’re right, I got lazy after the struggle to understand what was happening. I’ve changed it to sort by name, thanks!